xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/colorThemeInit.vm

```diff
@@ -48,7 +48,10 @@
 'notificationInfoColor'            : '#336699',
 'notificationSuccessColor'         : '#008000',
 'notificationWarningColor'         : '#D09000',
-'notificationErrorColor'           : '#CC3333'
+'notificationErrorColor'           : '#CC3333',
+'notificationEmailBg'              : '#f5f5f5',
+'notificationEmailInnerBg'         : '#ffffff',
+'notificationEmailTextColor'       : '#1d1f24'
```

Comment:
Why did you need to add new variables and could not reuse some of the existing values? 
for example: @notificationEmailBg could have used the @body-bg for example.

Reply:
Because we're talking about the email here, and it's not always ok to use the the general theme colors, for example if we have a dark themed wiki, we might not want the email to be dark as well.